webpack.config.js

```diff
@@ -230,16 +228,16 @@ module.exports = () => {
       // everything else out.
       //
       // https://webpack.js.org/api/module-methods/#import-<Paste>
-      new webpack.IgnorePlugin(
-        new RegExp(`/(?!(?:${syntaxes.join('|')}).js$).*js$`),
-        /brace[\\/]mode/,
-      ),
+      new webpack.IgnorePlugin({
+        resourceRegExp: new RegExp(`/(?!(?:${syntaxes.join('|')}).js$).*js$`),
+        contextRegExp: /brace[\\/]mode/
+      }),
 
       // Each time we change something, a new version of bundled assets is
       // produced. Since we use hash in filenames in order to invalidate cache
       // on change, we end up having multiple outdated copies in output
       // directory. Let's cleanup it before produce a fresh build.
-      new CleanWebpackPlugin([path.resolve(__dirname, 'dist')]),
```

Comment:
This shouldn't be removed?

Reply:
It seems it defaults now to `output.path` of the main webpack config which points to the same directory.